content/md/first-steps/manage-multidimensional-data-in-a-table.md

```diff
@@ -132,7 +132,17 @@ And if you are comfortable with the API, you can retrieve the table options with
 ## Import/export the table values of a product
 Exactly like for other attribute types, you can import and export your products with table attributes. But, as a table is a quite complexe value, the only way for the PIM to import table data among the other attributes of the product, is by translating your value into JSON in the "table" attribute column of your CSV or XLSX file.
 
-That's why **we highly recommend you to keep the import/export feature for data flows between the PIM and other systems purpose, and not for human enrichment purpose.**
+### Use the dedicated "product data table values" import/export jobs
+We know that, working with JSON format within a xlsx/csv file is not that much comfortable for manual enrichment. That's why we have created several import and export jobs in order to import and export the enriched table attributes, formatted as they are in the product edit form.  
```

Comment:
```suggestion
We know that working with JSON format within a xlsx/csv file is not that much comfortable for manual enrichment. That's why we have created several import and export jobs in order to import and export the enriched table attributes, formatted as they are in the product edit form.  
```

Reply:
```suggestion
There is another way, more readable and easier to configure using dedicated export and import jobs for the table attribute. It will only handle table attribute values and it will be formatted as they are in the product edit form.  
```